zone/spells.cpp

```diff
@@ -780,7 +779,12 @@ bool Mob::DoCastingChecksOnTarget(bool check_on_casting, int32 spell_id, Mob *sp
 			if (IsGroupSpell(spell_id)) {
 				return true;
 			} else if (spells[spell_id].target_type == ST_Self) {
-				spell_target = this;
+				if (IsBot() && (IsEffectInSpell(spell_id, SE_SummonCorpse) && RuleB(Bots, AllowCommandedSummonCorpse))) {
+					//spell_target = this;
+				}
+				else {
+					spell_target = this;
+				}
```

Comment:
?

Reply:
I need to change this and correct summon corpse for bots which was always broken. This was part of some testing that slipped through cleanup I think. I fixed it but looks like it might be broken again with a recent change